public/javascripts/xp.js

```diff
@@ -0,0 +1,36 @@
+/*
+ 1: 0 - 1000,
+ 2: 1000 - 3000,
+ 3: 3000 - 6000,
+ 4: 6000 - 10000,
+ etc...
+*/
```

Comment:
Please either provide more context for this comment, or delete it.

Reply:
I removed it.